Find a functionally equivalent code:android.ddm.DdmHandleProfiling.handleSPSS:COMMENT
Method Modifier: private     
Comment:/*
     * Handle a "Sample Profiling w/Streaming Start" request.
     */

@@ -3,9 +3,9 @@
     int bufferSize = in.getInt();
     int flags = in.getInt();
     int interval = in.getInt();
-    if (false) {
+    if (DEBUG) {
         Log.v("ddm-heap", "Sample prof stream start: size=" + bufferSize + ", flags=" + flags + ", interval=" + interval);
     }
     try {

